### ISSUE TITLE
Token

### ISSUE THREAD

**Issue by Stuunyez** [V4NSH4J/discord-mass-DM-GO#695]
How i can find discord token ? I need to create discord account and take token ?

**Comment by itschasa** [comment]
- create an account and login
- open devtools (ctrl shift i)
- open network panel
- click on one of the /science requests (if there isn't any, click around on discord until there is)
- scroll down to the request headers
- copy the value in the Authorisation header, that's your token

**Comment by Stuunyez** [comment]
Yes but in the tutorial he has many tokens so I create other discord accounts

**Comment by itschasa** [comment]
you can purchase tokens on discock

https://discock.app/
